### PR TITLE
Minor clarifications

### PR DESCRIPTION
**Pull request by ashcrow**
Various minor clarifications from going through the documentation so far.

### CODE REVIEW

File: beta-3-setup.md [hunk 1]
```diff
@@ -981,8 +987,7 @@ following as the `root` user:
     osc get node -o json | sed -e '/"resourceVersion"/d' > ~/nodes.json
 
 You will have the JSON output of the definition of all of your nodes. Go ahead
-and edit this file. Add the following to the beginning of the `"metadata": {}`
```
Comment (thoraxe) [inline beta-3-setup.md:984]: Was this sentence removed accidentally?

File: beta-3-setup.md [hunk 2]
```diff
@@ -3179,6 +3186,26 @@ wildcard space:
     foo.cloudapps.example.com 0 IN A 192.168.133.2
     ...
 
+### Resolving From Your Local Machine
+If you would like to use another system on the network to view and the OSE console you will need to
```
Comment (thoraxe) [inline beta-3-setup.md]: "to view and the OSE console" - this looks like a typo?

File: beta-3-setup.md [hunk 3]
```diff
@@ -3179,6 +3186,26 @@ wildcard space:
...
+
+    127.0.0.1       localhost.localdomain localhost
+    ::1     localhost6.localdomain6 localhost6
+    192.168.122.86 ose3-master.example.com
+
+
+#### Resolv
+Edit ```/etc/resolv.conf``` and add your ```ose3-master``` machine as the first nameserver.
+
+**Note**: If you are using NetworkManager your ```/etc/resolv.conf``` will change over time. If you find this an issue you can make the file immutable ```chattr +i /etc/resolv.conf```. Do not forget to run ```chattr -i /etc/resolv.conf``` after you are finished with these labs else NetworkManager won't be able to modify the file!
```
Comment (thoraxe) [inline beta-3-setup.md]: Wouldn't it be preferable to modify the dhclient configuration?

Also, adding "ose3-master" as the first nameserver would result in pointing at SkyDNS, which doesn't have DNS entries for the routes. If anything, you would want to be pointing at whatever node you're running dnsmasq on?

File: beta-3-setup.md (hunk 1 above)
Comment (ashcrow) [inline beta-3-setup.md:984]: Doh, yeah.

**Comment by ashcrow** [comment]
Removed the DNS portion and I believe I fixed the typos.

File: beta-3-setup.md [hunk 4]
```diff
@@ -1029,6 +1035,8 @@ Note: At release the user should not need to edit JSON like this; the
 installer should be able to configure nodes initially with desired labels,
 and there should be better tools for changing them afterward.
 
+Note: If you end up getting an error while attempting to update the nodes review your json. Ensure that there are commas in the previous element to your added label sections.
```
Comment (thoraxe) [inline beta-3-setup.md]: Should be:

If you end up getting an error while attempting to update the nodes, review your json.

**Pull request merged**